.idea/misc.xml

```diff
@@ -18,6 +18,8 @@
         <entry key="..\:/AndroidStudioProjects/Quotes/app/src/main/res/layout/share_action_option_pick_dialog.xml" value="0.1" />
         <entry key="..\:/AndroidStudioProjects/Quotes/app/src/main/res/layout/share_layout.xml" value="0.1" />
         <entry key="..\:/AndroidStudioProjects/Quotes/app/src/main/res/layout/tour_single_item.xml" value="0.1" />
+        <entry key="..\:/Users/hp/Desktop/Hacktober Fest/Quotes/app/src/main/res/drawable/ic_done.xml" value="0.2203125" />
+        <entry key="..\:/Users/hp/Desktop/Hacktober Fest/Quotes/app/src/main/res/layout/fragment_about.xml" value="0.20416666666666666" />
```

Comment:
Please revert

Reply:
Ok, but I have a doubt. Actually, there is a change in idea/misc.xml apart from these 2 changes so if I revert then that change will also get reverted, so I want to ask that should I just delete these two lines or is it necessary to revert the code?

Reply:
> Ok, but I have a doubt. Actually, there is a change in idea/misc.xml apart from these 2 changes so if I revert then that change will also get reverted, so I want to ask that should I just delete these two lines or is it necessary to revert the code?

@Devansh-gaur-1611 Um.. The contents of the file should be exactly same of what's in the source repo (i.e, this repo). If this is so, it wont show up in differences.